src/Util/GeometryUtil/GeometryUtil.js

```diff
@@ -211,6 +199,42 @@ class GeometryUtil {
     return multiGeom;
   }
 
+  /**
+   * Splits an array of geometries (and multi geometries) or a single MutliGeom
+   * into an array of single geometries.
+   *
+   * @param {ol.geom.Geometry|ol.geom.Geometry[]} geometries An (array of) ol.geom.geometries;
```

Comment:
An array of ol.geom.geometries or a single geometry, e.g. a multi_geometry.

Reply:
I think the more appropriate type would be: `ol.geom.SimpleGeometry` since we do not support collections, do we?